### DISCUSSION TITLE
force container update

### DISCUSSION THREAD

**Discussion by rogue73** [containrrr/watchtower#672] [Q&A] [answered]
Hi,
i have running some containers with docker and one of these is a watchtower container.
For one of the containers i have currently set the monitor-only label. Until here everything is as expected. Containers without the label are updated automaticlly and the container with this label is only logged with an info message "Found new ... image".
Now i'm searching for way to run watchtower with --run-once and ignoring this label to force the update.

Is there any possibility to force the update and ignore the monitor-only label?

Cheers,
Stefan

**Comment by simonaronsson** [comment, marked as answer]
Hi,

Currently, we lack support for this. If `monitor-only` has been set, you'll have to do the update yourself manually.

Best,
Simme

**Comment by spupuz** [comment]
how to force update without setting a label?

**Comment by tigerinus** [comment]
would love to have ability to `force update` a `monitor only` container.
